android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/PhysicalWebBroadcastService.java

```diff
@@ -62,7 +62,6 @@
     private NotificationManagerCompat mNotificationManager;
     private Handler mHandler = new Handler();
     private String mDisplayUrl;
```

Comment:
I've filed an issue in case you want to do it in a different change, but it'd be nice to get rid of this field as well.